src/hooks/searchEngines/useSolrSearchEngine.ts

```diff
@@ -105,6 +105,34 @@ export const useSolrSearchEngine = () => {
     return title;
   }, [map, replaceTemplates]);
 
+  const applyAttributesToFeature = useCallback((
+    olFeature: OlFeature,
+    dsResult: DataSearchResult
+  ): void => {
+
+    if (!map) {
+      return;
+    }
+
+    const blacklistedAttributes = [
+      'category',
+      'id',
+      'featureType',
+      'geometry',
+      'search',
+      '_version_'
+    ];
+
+    Object.keys(dsResult)
+      .filter(key => !blacklistedAttributes.includes(key))
+      .forEach(key => {
+        const value = dsResult[key];
+        if (value !== undefined) {
+          olFeature.set(key, value);
+        }
+      });
+  }, [map]);
```

Comment:
`map` doesn't seem to be required in the callback?

Reply:
Yes, I have removed `map` here.